crates/core/src/shared/transform.rs

```diff
@@ -132,10 +132,27 @@ impl<'a> JsxTransform {
         el: &ast::JSXElement<'a>,
         ctx: &mut TraverseCtx<'a>,
     ) -> TransformResult<'a> {
+        let tag_name = match el.opening_element.name {
+            ast::JSXElementName::Identifier(ref ident) => ident.name.clone(),
```

Comment:
```suggestion
        let tag_name = match &el.opening_element.name {
            ast::JSXElementName::Identifier(ident) => ident.name.clone(),
```
I believe this is more preferred in general